pkg/types/iface.go

```diff
@@ -30,3 +30,31 @@ func ValidateIfaceNames(ifaceList string) ([]string, error) {
 	}
 	return ifaces, nil
 }
+
+func ValidateAndSeparateFilters(ifaceList string) ([]string, []string, error) {
+	ifaces := strings.Split(ifaceList, ",")
```

Comment:
Nitpick: use a const for the delimiter, aka `const ifaceListDelimiter = ","`

Reply:
done